configuration/index.md

```diff
@@ -128,14 +130,37 @@ Every interface has it's own strong points.
   </tbody>
 </table>
 
-{: #text-vs-ui}
 ### Textual vs. Graphical Configuration
 
-With openHAB 1.x the smart home configuration was done via configuration files only.
-One of the most visible additions to openHAB 2 is the administrative web interface Paper UI.
+In openHAB 1.x, smart home configuration was done via configuration files only.
+openHAB 2 added the administrative web interface "Paper UI". It allows for configuration of the base system and setup of openHAB 2 add-ons.
+Please be aware that not every addon/binding of openHAB version 1.x has been updated to version 2 and thus to work with Paper UI.
+You can still run these but you need to use file configuration for them.
 
 Things and Items can either be defined and managed in configuration files or handled by Paper UI in a [system-side database]({{base}}/administration/jsondb.html).
-
-Both methods can be used **side-by-side or even mixed**, e.g. a Thing is discovered by Paper UI and linked Items are defined in a `.items` configuration file.
-
-Configuration done in files will be synced to the database but configuration done in the database will not be written to files.
+Both methods can be used **side-by-side or even mixed**, e.g. a Thing is discovered by Paper UI and linked Items are defined in a 
+`.items` configuration file.
+Configuration done in files will be synced to the database (so they are accessible in Paper UI) but configuration done in the database will not be written back to files, so if you restart openHAB, changes made in Paper UI won't survive and the files' configuration data will be used again.
```

Comment:
If i am right "the database" isn't introduces yet in the article, so this could be confusing for starters.
Maybe we should just mention, that it will be synced in background and point to the jsondb article?
Interested people could follow the link to jsondb and others could be perfextly fine with the information, that there is something synced in background.

Wdyt?

Reply:
yes. Do we have a jsondb article already ? didn't find any ...
Kai was also questioning if this (the contents) was right ?

Reply:
Yes we have a jsondb article:
https://www.openhab.org/docs/administration/jsondb.html

Reply:
the database was introduced in the previous sentence, and I forgot that the jsondb article also is already being linked to